wiki/People/The_Team/de.md

```diff
@@ -1,56 +1,103 @@
 ---
-outdated: true
+tags:
+  - osu! staff
+  - osu!team
+  - osu! team
+  - staff
+  - team osu!
 ---
 
 # Das Team
 
-## Gründer
+*Für eine Liste der Beförderungen und Rücktritte, siehe: [Mitarbeiter-Log](/wiki/Staff_Log)*
 
-| Name | Beschreibung |
+Die unten aufgeführten Personen bilden den Kern des **osu!-Teams** und sind hauptverantwortlich dafür, dass das Spiel für alle großartig ist.
+
+| Name | Rolle <!-- TODO: "Featured Artist outreach" isn't a role but I'm not sure how to write it as one... --> |
 | :-- | :-- |
-| [peppy](https://osu.ppy.sh/users/2) | Projektleiter |
+| ![][flag_AU] **[peppy](https://osu.ppy.sh/users/2)** | **Projektleiter** |
+| ![][flag_GB] [-Mo-](https://osu.ppy.sh/users/2202163) | Berater für die Modding- und Mapping-Szene |
+| ![][flag_CA] [Azer](https://osu.ppy.sh/users/2155578) | Organisator und Verwalter des World Cups |
+| ![][flag_US] [Chaos](https://osu.ppy.sh/users/2628870) | Moderation und Community-Berater |
+| ![][flag_US] [ChillierPear](https://osu.ppy.sh/users/9501251) | Organisator und Verwalter des World Cups |
+| ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | Projekt- und Community-Wartung, Featured Artist Kontaktaufnahme |
```

Comment:
```suggestion
| ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | Projekt- und Community-Betreuer, Featured Artist Kontaktaufnahme |
```
`Community-Wartung` sounds a bit wrong to me. It's like the community needs to be updated lol

Reply:
yeah true